src/main/java/in/twizmwaz/cardinal/module/repository/RepositoryModule.java

```diff
@@ -61,7 +61,7 @@
   public RepositoryModule() {
     //TODO: support alternate repos
     String mapRepository = Cardinal.getInstance().getConfig().getString("mapRepository");
-    File mapRepositoryRoot = new File(Paths.get(mapRepository).isAbsolute() ? mapRepository : Cardinal.getInstance().getDataFolder().getAbsolutePath() + mapRepository);
+    File mapRepositoryRoot = new File(Paths.get(mapRepository).isAbsolute() ? mapRepository : Cardinal.getInstance().getDataFolder().getAbsolutePath() + File.separator + mapRepository);
```

Comment:
This was missing the separator (it tried to default to plugins/cardinalrepo instead of /plugins/cardinal/repo)